boards.txt

```diff
@@ -897,8 +928,14 @@ GenF4.menu.xserial.none.build.xSerial=-DHAL_UART_MODULE_ENABLED -DHWSERIAL_NONE
 GenF4.menu.xserial.disabled=Disabled (no Serial support)
 GenF4.menu.xserial.disabled.build.xSerial=
 
-RAK.menu.xserial.generic=Enabled (generic 'Serial')
-RAK.menu.xserial.none=Enabled (no generic 'Serial')
+Sparky.menu.xserial.generic=Enabled with generic Serial
```

Comment:
Serial menu has been updated.
Could you align with this new definition:
https://github.com/stm32duino/Arduino_Core_STM32/pull/424/commits/8800a2deb89ae6381f700425673bdf590a8900be

Note that you can add USB also if needed